Assets/PatchKitTools/Editor/Menus/BuildAndPublish.cs

```diff
@@ -82,44 +81,69 @@ private void OnAppSelected(Api.Models.Main.App app)
             BeginBuildView();
         }
 
-        private void BeginBuildView()
+        private void BeginBuildView() // *** 3 view *** //
         {
-            var build = new Views.BuildApp();
+            var build = new Views.BuildApp(_selectedApp);
```

Comment:
Replace `var` with type name in places where used type is not obvious.

Reply:
But the type here was obvious...